.pre-commit-config.yaml

```diff
@@ -6,7 +6,6 @@ repos:
     rev: 20.8b1
     hooks:
       - id: black
-        language_version: python3.8
```

Comment:
Niet python3.9?

Reply:
Was about to say: _Nope. By default `black` uses the version of the Python interpreter it is running on_ But that only holds when you are running `black` from the command line in the project's Python environment. I don't now how `pre-commit` selects the Python interpreter for its own environment. So yes, we should specify it. Good catch :+1: